### ISSUE TITLE
org.keepassxc.appdata.xml contains invalid XML

### ISSUE THREAD

**Issue by magkopian** [keepassxreboot/keepassxc#1272]
During the process of building my package for the new release, Lintian gave me the following error.

```
E: keepassxc: appstream-metadata-invalid org.keepassxc.appdata.xml
N: 
N:    The specified AppStream metadata file does not consist of valid XML.
N:    
N:    Refer to https://wiki.debian.org/AppStream/Guidelines for details.
N:    
N:    Severity: serious, Certainty: certain
N:    
N:    Check: appstream-metadata, Type: binary
N: 
```

It appears that the `org.keepassxc.appdata.xml` contains invalid XML and as a result the build fails. I think that the problem is with the comment in the second line being,

`<!-<li>Copyright 2017 KeePassXC Team <team@keepassxc.org> -->`

instead of,

`<!-- <li>Copyright 2017 KeePassXC Team <team@keepassxc.org> -->`

By the way, I've built the package anyway by configuring Lintian to not fail the build, since I knew the reason it was failing wasn't a real problem.

**Comment by TheZ3ro** [comment]
Yup we have seen this. Thanks for the report! We will fix ASAP

**Comment by phoerious** [comment]
Fixed in #1273
